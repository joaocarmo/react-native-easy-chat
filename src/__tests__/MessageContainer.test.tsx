import renderer from 'react-test-renderer'
import { MessageContainer } from '../EasyChat'

it('should render <MessageContainer /> and compare with snapshot', () => {
  const tree = renderer.create(<MessageContainer />).toJSON()

  expect(tree).toMatchSnapshot()
})
