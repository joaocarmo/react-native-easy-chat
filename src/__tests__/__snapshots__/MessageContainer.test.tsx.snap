// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <MessageContainer /> and compare with snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RCTScrollView
    ListEmptyComponent={[Function]}
    ListFooterComponent={[Function]}
    ListHeaderComponent={[Function]}
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Object {
        "flexGrow": 1,
        "justifyContent": "flex-start",
      }
    }
    data={Array []}
    enableEmptySections={true}
    extraData={
      Array [
        null,
        false,
      ]
    }
    getItem={[Function]}
    getItemCount={[Function]}
    invertStickyHeaders={true}
    inverted={true}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onEndReached={[Function]}
    onEndReachedThreshold={0.1}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={100}
    stickyHeaderIndices={Array []}
    style={
      Array [
        Object {
          "transform": Array [
            Object {
              "scaleY": -1,
            },
          ],
        },
        Object {
          "flex": 1,
        },
      ]
    }
    viewabilityConfigCallbackPairs={Array []}
  >
    <View>
      <View
        onLayout={[Function]}
        style={
          Object {
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          }
        }
      >
        <View
          collapsable={false}
          style={
            Object {
              "backgroundColor": "#f0f0f0",
              "borderRadius": 15,
              "height": 0,
              "marginBottom": 0,
              "marginLeft": 8,
              "transform": Array [
                Object {
                  "translateY": 200,
                },
              ],
              "width": 45,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flex": 1,
          }
        }
      />
      <View
        onLayout={[Function]}
        style={
          Object {
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        />
      </View>
    </View>
  </RCTScrollView>
</View>
`;
